packages/core/plugins/core-plugins-server-internal/index.ts

```diff
@@ -13,3 +13,4 @@ export type {
   DiscoveredPlugins,
   PluginDependencies,
 } from './src';
+export { PluginSystemOverrides } from './src/plugin_system_overrides';
```

Comment:
NIT: consistency - export from `src/index`, then re-export from `./src` in this file as for everything else.